langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -100,10 +108,11 @@ public AzureOpenAiChatModel(OpenAIClient client,
                                 List<AzureChatExtensionConfiguration> dataSources,
                                 AzureChatEnhancementConfiguration enhancements,
                                 Long seed,
-                                ChatCompletionsResponseFormat responseFormat,
+                                ResponseFormat responseFormat,
+                                boolean strictJsonSchema,
```

Comment:
We need to not forget to implement `strictTools` as well, it should be pretty straightforward.

Reply:
I reverted to the former API

Reply:
Just in case: my comment was about `strictTools`, not `strictJsonSchema`, it is slightly different feature. Should we create a separate PR for `strictTools`?

Reply:
Oh sorry, I thought you made a typo! I haven't had a look at `strictTools`